### ISSUE TITLE
How to pass the testing samples to the model?

### ISSUE THREAD

**Issue by vuhtr** [clks-wzz/FAS-SGTD#27]
Hi there

In your paper, you write that "...For the final classification score, we feed the sequential frames into the network and obtain depth maps and the living logits".
However, I can't find that part of code in the file `generate_data_test.py`. Everything this file does is almost the same for training: we need to pass a scene image (video's frame) and its depth map, even when testing (it's seem unreasonable).

Have I misunderstood somewhere? Could you please point this detail out for me...

Thank you very much in advance!
